projects/lib/src/admin/components/page-list/page-list.component.ts

```diff
@@ -24,6 +28,7 @@ import DEFAULT_ASSET_TYPES, {
   ASSET_TYPES,
 } from '../../constants/asset-types.constants';
 import { PageContentDoc } from '../../models/page-content-doc.interface';
+import { FormBuilder, FormGroup } from '@angular/forms';
```

Comment:
it look these are no longer being used in this component, can you remove this line please?